.ci/test_windows.ps1

```diff
@@ -96,8 +96,8 @@ elseif ($env:TASK -eq "bdist") {
   cd $env:BUILD_SOURCESDIRECTORY
   sh "build-python.sh" bdist_wheel --integrated-opencl ; Check-Output $?
   sh $env:BUILD_SOURCESDIRECTORY/.ci/check_python_dists.sh $env:BUILD_SOURCESDIRECTORY/dist ; Check-Output $?
-  cd dist; pip install --user @(Get-ChildItem *.whl) ; Check-Output $?
-  cp @(Get-ChildItem *.whl) $env:BUILD_ARTIFACTSTAGINGDIRECTORY
+  cd dist; pip install --user @(Get-ChildItem *py3-none-win_amd64.whl) ; Check-Output $?
+  cp @(Get-ChildItem *py3-none-win_amd64.whl) $env:BUILD_ARTIFACTSTAGINGDIRECTORY
```

Comment:
This ensures we get a loud error if the wheels are not tagged in the way we'd expect.

Since as far as I can tell, you can't pass a `--plat-name` through `python -m build` the way you could with  `python setup.py bdist_wheel --plat-name=win-amd64`.

To be clear... `scikit-build-core` is doing exactly the right thing here and producing the expected tags. This is just a precaution to help us detect if that changes, before publishing mistagged wheels to PyPI (e.g. like #4788).